pom.xml

```diff
@@ -36,8 +36,8 @@
         </plugin> 
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.5</version>
+        <artifactId>maven-plugin-plugin</artifactId>
```

Comment:
`maven-plugin-plugin`? We will want the `maven-release-plugin` for creating release candidates